pom.xml

```diff
@@ -48,7 +49,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <cdap.version>3.2.1</cdap.version>
```

Comment:
Let's not make this change. Let's depend on 3.2.1 and make the relevant changes we need to the plugin.